compose/cli-command.md

```diff
@@ -0,0 +1,14 @@
+---
+description: Compose in the Docker CLI Tech Preview
+keywords: documentation, docs, docker, compose, containers
+title: Compose in the Docker CLI Tech Preview
+---
+
```

Comment:
Suggest a note here that states this is a tech preview and it must not be used in production environments.

> Important
>
> The `compose` command  in the Docker CLI is currently available as a Tech Preview. We recommend that you do not use this in production environments.
>
> Your feedback is important to us. Let us know your feedback on the new 'compose' command by creating an issue in the [Compose-CLI](https://github.com/docker/compose-cli/issues){:target="_blank" rel="noopener" class="_"} GitHub repository.
{: .important}